app/views/posts/index.html.erb

```diff
@@ -1 +1,32 @@
-<h1>this is posts</h1>
+<% identifier = @posts.first %>
+<% user = User.find(id = identifier.author_id) %>
+<div class="card">
+  <div class="photo">
+    <%= image_tag(user.photo, width: "150px") %>
+  </div>
+  <div class="user_card">
+    <div class="user_name"><%= user.name %></div>
+    <div class="user_posts">Number of posts: <%= user.posts_counter %></div>
+  </div>
+</div>
+
+<% @posts.each do |post| %>
+  <div class="post">
+    <div class="post_title"><%= post.title %></div>
+    <div class="post_text"><%= post.text %></div>
+    <div class="post_media">
+      <div class="post_comments">Comments: <%= post.comments_counter %></div>
+      <div class="post_likes">Likes: <%= post.likes_counter %></div>
+    </div>
+  </div>
```

Comment:
- [ ] Clicking on each post should take me to the post-show page. Please add a click event for that.

Reply:
Hi @whiz25 Thank you for reviewing my code.
About your requested change, I applied exactly what the previous reviewer said that the link should take us to the post-show page and it does I tried your changes to apply and the link does not work. 
here is the routes


![Screenshot from 2022-08-19 18-51-30](https://user-images.githubusercontent.com/93380132/185641702-fb332df1-e503-48fa-9704-0e5c27cbd4cf.png)


 And the browser URL is here http://127.0.0.1:3000/users/2/posts/2
from my side, everything is working as expected.
I would be happy if you leave your opinion. and make me clear if I am wrong.